UM/Settings/DefinitionContainer.py

```diff
@@ -333,16 +333,21 @@ def deserialize(self, serialized: str, file_name: Optional[str] = None) -> str:
         self._metadata["container_type"] = DefinitionContainer
 
         self._deserializeDefinitions(parsed["settings"])
+
+        for definition in self._definitions:
+            self._updateRelations(definition)
```

Comment:
Are we here also just interested in the newly added definitions?
```suggestion
        added_definitions = self._deserializeDefinitions(parsed["settings"])

        for definition in added_definitions:
            self._updateRelations(definition)
```

Reply:
Well I've been thinking about that indeed. The result should be similar, but I preferred to keep the old code to avoid breaking something. Could you test this suggestion during component test ?